platform/android/MapboxGLAndroidSDKTestApp/src/androidTest/java/com/mapbox/mapboxsdk/testapp/style/SymbolLayerTest.java

```diff
@@ -1004,7 +1005,6 @@ public void testIconHaloColorAsExpression() {
     });
   }
 
-
```

Comment:
Nit: can we get rid of this diff? I looked at layer.junit.ejs and couldn't actually see where it was coming from..

Reply:
It's hard to tell which change is it, and since it actually fixes the formatting of the resulting files, I'm going to keep it in.